include/seqan3/alphabet/union_alphabet.hpp

```diff
@@ -44,40 +44,16 @@
 #include <optional>
 #include <cassert>
 
-#include "nucleotide/dna5.hpp"
 #include "alphabet.hpp"
 
 /*! The union alphabet
  * \ingroup alphabet
  */
 
-namespace seqan3
-{
+#include <algorithm>
 
-namespace detail
+namespace seqan3::detail
 {
-template <typename tuple_t, size_t ...I>
-constexpr auto alphabet_objects(std::index_sequence<I...>)
-{
-    using variant_t = std::variant<std::tuple_element_t<I, tuple_t>...>;
-    using list_t = std::initializer_list<variant_t>;
-    return list_t
-    {
-        variant_t
-        {
-            std::in_place_index_t<I>{},
-            std::tuple_element_t<I, tuple_t>{}
-        }...
-    };
-}
-
-template <typename tuple_t, size_t ...I>
-constexpr auto alphabet_sizes(std::index_sequence<I...>)
-{
-    using integral_t = uint8_t;
-    using list_t = std::initializer_list<integral_t>;
-    return list_t{std::tuple_element_t<I, tuple_t>::value_size...};
-}
 
 template <size_t size>
 constexpr auto at_least_type() {
```

Comment:
open brace to newline. Also the values should not be `4294967295u`, but `4294967295ull`, otherwise it will already be truncated. Also take `unsigned long long` as the template type. Also think about making it a `using` decleration with std::conditional, as it is more intuitive than having to call decltype on this.
See the style guide on how to write type metafunctions...